db/maintenance/list.go

```diff
@@ -0,0 +1,56 @@
+package maintenance
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/olekukonko/tablewriter"
+	"golang.org/x/text/cases"
+	"golang.org/x/text/language"
+	"gopkg.in/errgo.v1"
+
+	"github.com/Scalingo/cli/config"
+	"github.com/Scalingo/cli/io"
+	"github.com/Scalingo/cli/utils"
+	"github.com/Scalingo/go-scalingo/v6"
+)
+
+func List(ctx context.Context, app string, addonName string, paginationOpts scalingo.PaginationOpts) error {
+	caser := cases.Title(language.English)
+	c, err := config.ScalingoClient(ctx)
+	if err != nil {
+		return errgo.Notef(err, "get Scalingo client")
```

Comment:
We should use `Wrap` now, according to our [doc](https://www.notion.so/scalingooriginal/Go-Error-Management-1ca42114c730415e95755027935eaccc?pvs=4#e9785e8022e5415eb211e966deefcea2)